src/observer/json_rows.py

```diff
@@ -0,0 +1,28 @@
+from collections import OrderedDict
+import itertools
+from . import utils
+from django.http import StreamingHttpResponse
+
+def streaming_response(filename, rows):
+    body = []
+    body.append((utils.safe_json_dumps(row) + "\n") for row in rows)
+    response = StreamingHttpResponse(itertools.chain.from_iterable(body), content_type="application/json")
+    response['Content-Disposition'] = 'attachment; filename="%s.rows.json"' % filename
```

Comment:
nothing strange at the HTTP headers level

Reply:
Observer isn't following the REST model like the other services, it was never integrated properly. It might even be replaced entirely if BQ or something feeding off of BQ can return RSS feeds. Observer is very nice for querying article data but it just never caught on